modules/economy-system/commands/economy-system.js

```diff
@@ -0,0 +1,298 @@
+const {balance, createleaderboard} = require('../economy-system');
+const {embedType} = require('../../../src/functions/helpers');
+
+module.exports.beforeSubcommand = async function (interaction) {
+    interaction.str = interaction.client.configurations['economy-system']['strings'];
+    interaction.config = interaction.client.configurations['economy-system']['config'];
+};
+
+/**
+ * Function to handle the cooldown stuff
+ * @private
+ * @param {string} command The command
+ * @param {integer} duration The duration of the cooldown (in ms)
+ * @param {userId} userId Id of the User
+ * @returns {Promise<boolean>}
+ */
+function cooldown (command, duration, userId) {
+    const model = interaction.client.models['economy-system']['cooldown'];
+    const cooldownModel = model.findOne({
+        where: {
+            userId: userId,
+            command: command
+        }
+    });
+    if (cooldownModel) {
+        // check cooldown duration
+        if (cooldownModel.timestamp.getTime() + duration > Date.now()) return false;
+        cooldownModel.timestamp = new Date();
+        cooldownModel.save();
+        return true;
+    } else {
+        // create the model
+        model.create({
+            userId: userId,
+            command: command,
+            timestamp: new Date()
+        });
+        return true;
+    }
+}
+
+module.exports.subcommands = {
+    'work': async function (interaction) {
+        if (!cooldown('work', interaction.config['workCooldown'] * 60000, interaction.user.id)) return interaction.reply(embedType(interaction.str['cooldown'], {}, { ephemeral: true }));
+        const moneyToAdd = Math.floor(Math.random() * (interaction.config['maxWorkMoney'] - interaction.config['minWorkMoney'])) + interaction.config['minWorkMoney'];
+        balance(interaction.client, interaction.user.id, 'add', moneyToAdd);
+        interaction.reply(embedType(interaction.str['workSuccess'], {'%earned%': `${moneyToAdd} ${interaction.config['currencySymbol']}`}, { ephemeral: true }));
+        createleaderboard(interaction.client);
+        interaction.client.logger.info(`[economy-system] The user ${interaction.user.username}#${interaction.user.discriminator} gained ${moneyToAdd} ${interaction.config['currencySymbol']} by working`);
+        if (interaction.client.logChannel) interaction.client.logChannel.send(`[economy-system] The user ${interaction.user.username}#${interaction.user.discriminator} gained ${moneyToAdd} ${interaction.config['currencySymbol']} by working`);
+    },
+    'crime': async function (interaction) {
+        if (!cooldown('work', interaction.config['crimeCooldown'] * 60000, interaction.user.id)) return interaction.reply(embedType(interaction.str['cooldown'], {}, { ephemeral: true }));
```

Comment:
Wrong cooldown type here (and also in the following subcommands)
```suggestion
        if (!cooldown('crime', interaction.config['crimeCooldown'] * 60000, interaction.user.id)) return interaction.reply(embedType(interaction.str['cooldown'], {}, { ephemeral: true }));
```